### ISSUE TITLE
Suggestion: [Towny Camps] LWC Compatibility 

### ISSUE THREAD

**Issue by Litning11** [TownyAdvanced/Towny#5368]
A small suggestion - currently LWC does have a Towny integration that recognises Towny claims and has the option to only allow blocks to be locked within this claim. This system *technically* works for Camps too but it's not very clear.
Chests placed within a camp will not be locked by LWC, only by Towny denying interaction. It's confusing for new players (the target demographic of Camps) when they get notified their chests aren't locked. This also causes problems where chests can be accessed by anyone with interaction perms within the camp.

Tl;dr - Make LWC recognise Camp claims and have them work the same way they do with Towns.
(If this is an issue on LWC please tell me and I'll direct this issue at them)

Thanks.
